### DISCUSSION TITLE
Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230822

### DISCUSSION THREAD

**Discussion by brkirch** [brkirch/stable-diffusion-webui#35] [Announcements]
This is an offline standalone installer for a modified version of AUTOMATIC1111's Stable Diffusion web UI that includes experimental changes and uses the nightly PyTorch build from 8/21. Note that this is an test build so crashes and bugs should be expected. All dependencies needed to start using Stable Diffusion (other than the model) are included and this does not require Xcode command line tools to be installed.

This release currently only has the installer for Apple silicon Macs, an installer for Intel Macs will be included with 1.6.0 RC. macOS 13.4 or later is highly recommended, older versions of macOS are untested.

Python has been updated to 3.10.12 and the `--electron` option from @diegocr's PR (AUTOMATIC1111/stable-diffusion-webui#12694) has been added.

<hr /><em>This discussion was created from the release <a href='https://github.com/brkirch/stable-diffusion-webui/releases/tag/20230822_experimental'>Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230822</a>.</em>

**Comment by iliavls** [comment]
Thank you very much for your job!

**Comment by RobbieLiu** [comment]
Intel处理器提示安装不了

**Comment by AlaaFlashi** [comment]
please remove the `--electron` option, it gives no extra advantage and limiting the use to a single tab only making it hard to mange screen space
also mac os is giving an annoying message every time generation is complete to allow notification for electron (in that case http://127.0.0.1:7860) and i went to setting gave the permission for notification to electron in every thing but still getting this message every time, very annoying
please revert back to `--autolaunch`

**Comment by RobbieLiu** [comment]
Why？how install?
This is for Apple silicon Macs only, it will not work on Intel processors.
